packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/ProjectCard.js

```diff
@@ -1,117 +1,171 @@
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
     max-width: 100%;
     max-height: 100%;
   }
 `;
 
-const FullWidthRow = styled.div`
-  grid-column: 1 / -1;
-  padding: 0 16px;
+const Title = styled(Typography)`
+  font-size: 1.25rem;
+  line-height: 1.4;
+  font-weight: 500;
+  margin-bottom: 0.625rem;
 `;
 
-const Countries = styled(FullWidthRow)`
-  margin-bottom: 8px;
-  font-size: 14px;
-  opacity: 0.7;
-  padding: 0;
+const Text = styled(Typography)`
+  font-size: 0.875rem;
+  line-height: 1.2;
+  margin-bottom: 0.625rem;
 `;
 
-const Footer = styled(FullWidthRow)`
-  display: grid;
-  align-content: end;
-  padding: 0 16px;
+const Body = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
 `;
 
-export const ProjectCard = ({ name, description, imageUrl, logoUrl, names, projectButton }) => {
-  return (
-    <Card>
-      <Header backgroundImage={imageUrl}>
-        {logoUrl && (
-          <Logo>
-            <img alt="project logo" src={logoUrl} />
-          </Logo>
-        )}
-      </Header>
+const BLUE = '#1978D4';
+
+const StyledButton = styled(Button)`
+  background: ${BLUE};
+  border: 1px solid ${BLUE};
+  color: white;
+  border-radius: 3px;
+  font-weight: 500;
+  font-size: 0.875rem;
+  line-height: 1.2;
+  text-transform: none;
+  padding: 0.6875rem 1.25rem;
+  min-width: 11.5rem;
+
+  &:hover {
+    background: ${darken(BLUE, 0.1)};
+  }
+
+  .MuiSvgIcon-root {
+    font-size: 1.2em;
+  }
+`;
+
+const OutlineButton = styled(StyledButton)`
+  border: 1px solid ${BLUE};
+  color: ${BLUE};
+  background: transparent;
+
+  &:hover {
+    background-color: rgba(255, 255, 255, 0.08);
+  }
+`;
+
+// eslint-disable-next-line react/prop-types
+export const ProjectDeniedButton = ({ onClick }) => (
+  <OutlineButton onClick={onClick} startIcon={<Lock />}>
+    Request access
+  </OutlineButton>
+);
+
+export const ProjectLoginButton = ({ onClick }) => (
+  <OutlineButton onClick={onClick}>Log in</OutlineButton>
+);
+
+// eslint-disable-next-line react/prop-types
+export const ProjectPendingButton = ({ onClick }) => (
+  <OutlineButton onClick={onClick} startIcon={<Alarm />}>
+    Approval in progress
+  </OutlineButton>
+);
+// eslint-disable-next-line react/prop-types
+export const ProjectAllowedButton = ({ onClick }) => (
+  <StyledButton onClick={onClick}>View project</StyledButton>
+);
+
+function getCountryNames(projectName, countryNames) {
+  if (projectName === 'Disaster Response') {
+    return 'Global';
+  }
+
+  if (countryNames.length < 3) {
+    return countryNames.sort().join(', ');
+  }
+
+  return 'Multiple countries';
+}
+
+function getDescription(text, limit = 190) {
+  return text.length > limit ? `${text.substring(0, limit)}...` : text;
+}
+
+export const ProjectCard = ({ name, description, logoUrl, names, ProjectButton }) => (
+  <Card>
+    {logoUrl && (
+      <Logo>
+        <img alt="project logo" src={logoUrl} />
```

Comment:
This is very minor, but could we make the `alt` text here like `${name} logo` so that it is representative of the project? Makes it more meaningful to assistive technology users.